src/api/index.py

```diff
@@ -36,10 +36,13 @@ async def root():
 
 @app.post("/analyze_video")
 async def analyze_video(
-    video_url: str, target_item: str
+    video_url: str = "https://youtu.be/MNn9qKG2UFI", target_item: str = "car"
```

Comment:
I like this. I would instead import the `default_variables` module and then use the `video_url` as the value for `video_url`. Something like this:

```python
from src.utils import default_variables as dv
from typing import Optional

...
video_url: Optional[str]=dv.video_url, target_item: Optional[str]=dv.target_item
```

, while also adding the `target_item` to `default_variables`.

Reply:
I like that plan!

Reply:
Thanks for updating the code!